### ISSUE TITLE
Mod refringe-customraidtimes package.json main property points to non-existing file

### ISSUE THREAD

**Issue by GameAreaCZ** [refringe/CustomRaidTimes#9]
### Discord Username

GameArea.cz

### What happened?

Mod is not working.

### SPT Version

3.8.0

### Server Log

```shell
Mod refringe-customraidtimes package.json main property points to non-existing file
```
and when I go to game for example factory still 20 minutes. And I used the override for all to 360 minutes.

**Comment by drmahorin** [comment]
Can confirm that, installed as usual

**Comment by drmahorin** [comment]
UPD. @GameAreaCZ, `package.json` states that main is `CustomRaidTimes.js` but it's `customRaidTimes.js` instead so in case sensitive systems it will fail

You need to rename file into `CustomRaidTimes.ts`

**Comment by refringe** [comment]
Resolved in [v1.7.0](https://github.com/refringe/CustomRaidTimes/releases/tag/v1.7.0)
